src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java

```diff
@@ -98,4 +100,21 @@ public void shouldNotLogLongLinks() throws Exception {
         verify(checkConfig, getPath("InputLineLengthLongLink.java"), expected);
     }
 
+    @Test
+    public void countUnicodePointsOnce() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(LineLengthCheck.class);
+        checkConfig.addAttribute("max", "100");
+        // we need to set charset to let test pass on windows where default charset is not UTF-8
+        System.setProperty("file.encoding", StandardCharsets.UTF_8.name());
```

Comment:
I don't this is acceptable as it will clash with all other tests and not just this one. You don't even unset the property so you will be leaking into other tests. Even if you do unset, your not guaranteed your code will hit the unset even in the same test.

This has to be passed into the configuration object and set via checker.

Reply:
I didn't feel good about this either, thanks suggesting an alternative.

Reply:
Fixed.